anesthetic/samples.py

```diff
@@ -413,8 +413,8 @@ def plot_2d(self, axes=None, *args, **kwargs):
         'kde': {'diagonal': 'kde_1d', 'lower': 'kde_2d'},
         'kde_1d': {'diagonal': 'kde_1d'},
         'kde_2d': {'lower': 'kde_2d'},
-        'fastkde': {'diagonal': 'fastkde_1d', 'lower': 'fastkde_2d'},
         'hist': {'diagonal': 'hist_1d', 'lower': 'hist_2d'},
+        'fastkde': {'diagonal': 'fastkde_1d', 'lower': 'fastkde_2d'},
```

Comment:
I prefer the old ordering: kdes, fastkde, hists.
I think this comes from the already merged #314.

Reply:
mismerge -- fixed it cbab943